### ISSUE TITLE
5.4.0 is packaged with a broken version of MathJax

### ISSUE THREAD

**Issue by markhkim** [jupyter/notebook#3299]
A fresh installation of Jupyter (5.4.0) via `pip install jupyter` on Debian stretch is incapable of rendering any LaTeX formula, even the simplest ones. All instances of LaTeX formulas show `[Math processing error]`. Overwriting the `notebook/static/components/MathJax` directory with the MathJax package from mathjax.org solves the issue, so I suspect the MathJax package included in Jupyter notebook version 5.4.0 is broken.

**Comment by takluyver** [comment]
I've just done `pip install notebook` in a fresh environment, and mathjax seems to be working in both Firefox and Chromium.

Did you have any version of Jupyter installed before? It's possible that there is something in your browser's cache that affected it.

**Comment by markhkim** [comment]
Strange, I had cleared my cache on Chrome and all and the problem had persisted. But then, I just checked with venv and Firefox, and MathJax works just fine, as you said.

**Comment by takluyver** [comment]
We've noticed that sometimes stuff seems to linger even when we clear the cache. We've got a few tricks to try when we suspect that:

- Hit Ctrl-F5 several times in quick succession. Bizarrely, this sometimes seems to clear things that just pressing it once doesn't.
- Refresh with the developer tools open
- Open it in private browsing mode
- Open it in another browser
- Run the server on a different port (add e.g. `--port 8567` to the command line)
